README.md

```diff
@@ -190,6 +190,10 @@ PUT Body (optional):
 + list local charts
     - `GET`
     - `/api/charts/upload`
+    
++ helm version --template='version: {{.Version}} go_version: {{.GoVersion}}'
```

Comment:
why is this template specified?

Reply:
Because Christian filtered the returned value of the function with: chartutil.DefaultCapabilities.HelmVersion, so I adjusted the result with the template to give the same output. https://github.com/helm/helm/blob/main/pkg/chartutil/capabilities.go
Here e.g. the response:
{
    "code": 0,
    "data": {
        "version": "v3.6",
        "go_version": "go1.17.7"
    }
}

Reply:
Maybe we could add in the response instead?